src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationemployment/api/integration/resource/SARReadinessProfileGetShould.kt

```diff
@@ -66,17 +69,86 @@ class SARReadinessProfileGetShould : SARReadinessProfileTestCase() {
     assertAddReadinessProfileIsOk(prisonNumber, profileRequestOfKnownPRN)
     val expectedProfile = profileJsonOfKnownPRN
 
-    val sarResult = assertGetSARResponseIsOk(prn = prisonNumber)
+    val sarResult = assertGetSARResponseIsOk(expectedProfileAsJson = expectedProfile, prn = prisonNumber)
 
     assertThat(sarResult.body).isNotNull
     val json = objectMapper.readTree(sarResult.body!!.asJson())
     val jsonContent = json.findPath("content")
     assertThat(jsonContent.isMissingNode).isFalse()
     assertThat(jsonContent.get("offenderId").textValue()).isEqualTo(prisonNumber)
     assertThat(jsonContent.get("bookingId").longValue()).isEqualTo(bookingIdOfKnownPRN)
-    assertThat(jsonContent.get("profileData"))
-      .usingRecursiveComparison()
-      .ignoringFieldsMatchingRegexes(".*createdBy.*", ".*createdDateTime.*", ".*modifiedBy.*", ".*modifiedDateTime.*")
-      .isEqualTo(expectedProfile)
+  }
+
+  @Test
+  fun `reply 200(OK), when requesting a SAR with specified period`() {
```

Comment:
Is it `with` or `within`?

Reply:
I meant with (with a few conditions), just this time it's a period), I didn't meant the profile is within a period; Our data is not accurately "within a period" (WR lacked data snapshot at specific time or time period)

Reply:
What I tried to achieve with "specified period" parameters, is just to reduce data points that are more obviously beyond the time frame.